packages/app/components/user/UserDataCard.tsx

```diff
@@ -147,7 +147,7 @@ const UserDataCard = ({
                   // marginTop: '-3px',
                 }}
               >
-                Total Weight: {total_weight.toFixed(2)}g
+                Total Weight: {total_weight?.toFixed(2)}
```

Comment:
I think we need a unit that represents the weight here. it can be gram or may be we can get it from user profile and set the preferred unit. either way

Reply:
yeah i checked the unit not coming from anywhere in this so i, added the static g again here